weed/filer/ydb/readme.md

```diff
@@ -0,0 +1,26 @@
+## YDB
+
+database: https://github.com/ydb-platform/ydb
+
+go driver: https://github.com/ydb-platform/ydb-go-sdk
+
+options:
+
+```
+[ydb]
```

Comment:
Need to add a section to filer.toml example to scaffold.

Also, how much larger after this PR? If it is too large, e.g., more than 5MB larger, we may need to do something similar to etcd to compile it conditionally.

Reply:
> Need to add a section to filer.toml example to scaffold.
> 
> Also, the compiled binary will be how much larger after this PR? If it is too large, e.g., more than 5MB larger, we may need to do something similar to etcd to compile it conditionally.

without ydb store
```
 ls -lh weed 
-rwxr-xr-x  1 lebedev_k  staff    74M May  3 23:15 weed
```

with ydb store
```
-rwxr-xr-x  1 lebedev_k  staff    78M May  3 22:57 weed
```

exactly 5 mb, like looking into the water :)

add build tag